.github/workflows/docker.yml

```
@@ -0,0 +1,57 @@
+          password: ${{ secrets.GITHUB_TOKEN }}
+          logout: true
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
```

actionlint (1.7.12)

[error] the runner of "docker/metadata-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)